### PR TITLE
HOCS-3087 Updating dockerfile and adding health check file

### PR DESCRIPTION
**Pull request by gardnersam**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,6 +25,6 @@ RUN chmod a+x /app/scripts/*
 
 EXPOSE 8080
 
-USER ${USER_ID}
+USER ${USER}
 
-CMD /app/scripts/run.sh
\ No newline at end of file
+CMD ["sh", "/app/scripts/run.sh"]
```
Comment (LiamMacP) [inline Dockerfile:30]: Can we add a new line to the bottom of this?

File: scripts/HealthCheck.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+/**
+ * Class used for container health check by docker compose
+ */
+public class HealthCheck {
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+        var client = HttpClient.newHttpClient();
+        var request = HttpRequest.newBuilder()
+                .uri(URI.create("http://localhost:8080/actuator/health"))
```
Comment (LiamMacP) [inline scripts/HealthCheck.java:17]: Same here, could we use `server.port`? https://github.com/UKHomeOffice/hocs-templates/blob/a16384322916d2ffd615ce23c26c8dccb38cf76e/src/main/resources/application.properties#L5

File: Dockerfile (hunk 1 above)
Comment (gardnersam) [reply Dockerfile]: Yep done

File: scripts/HealthCheck.java (hunk 2 above)
Comment (gardnersam) [reply scripts/HealthCheck.java]: This file now uses a command line argument to set the end point to check so it is fully customisable

**Review by philbarton — APPROVED** [review]
Looks good to me

**Pull request merged**